db/seeds.rb

```
@@ -22,9 +25,29 @@ def seed_station_database(file_path)
   stations.each do |station|
     city = City.find_by(city: station[:city])
     station.delete_if { |k,v| k == :lat || k == :long || k == :city}
-    city.stations.create(station)
+    city.stations.create!(station)
```

Just curious, what's the ! do?

---

From what I read, the ! in create will raise an exception instead of just returning false if, in the event, the creation fails.

---

Interesting, good to know :+1: